src/logic/safe/store/actions/createTransaction.ts

```diff
@@ -159,7 +159,7 @@ export class TxSender {
             txArgs.sigs,
           )
           .encodeABI()
-      : txHash && safeInstance.methods.approveHash(txHash).encodeABI()
+      : this.txHash && safeInstance.methods.approveHash(this.txHash).encodeABI()
```

Comment:
Why this change? I see that before we were destructuring `txHash` from `this`

Reply:
`txHash` wasn't updating when destructured